packages/SketchMorph2-Core.package/M2Backend.class/instance/chooseFont.st

```diff
@@ -1,3 +1,4 @@
 tool selection
 chooseFont
+	self currentCursor: nil.
```

Comment:
This probably leads to the problem that, after choosing the font, no context cursor is shown.